src/routes/about.tsx

```diff
@@ -1,10 +1,48 @@
 import { NavigateFunction, useNavigate } from "react-router-dom"
+import type { GradeDistribution } from "~components/ProfileGrades";
+
+export interface ProfessorProfileInterface {
+  name: string;
+  profilePicUrl: string;
+  rmpScore: number;
+  diffScore: number;
+  wtaScore: number;
+  rmpTags: string[];
+  gradeDistributions: GradeDistribution[];
+  ratingsDistribution: number[]; // temp
+}
 
 export const About = () => {
   const navigation: NavigateFunction = useNavigate()
 
+  const mockData: ProfessorProfileInterface = {
```

Comment:
Always separate your mock data from the place where you are using it! This helps make sure you are passing variables to components the right way.